odoo/addons/base/models/res_lang.py

```diff
@@ -351,6 +369,17 @@ def action_activate_langs(self):
             }
         }
 
+    def name_get(self):
+        """ Add to language name a '(not installed)' when it is not active """
+        res = []
+        display_uninstalled_suffix = self.env.context.get('lang_uninstalled')
+        for lang in self:
+            lang_name = lang.name
+            if not lang.active and display_uninstalled_suffix:
+                lang_name += _(" (not installed)")
```

Comment:
use a placeholder for translations https://www.odoo.com/documentation/15.0/developer/howtos/translations.html#blocks
(not all languages have the same word orders)